ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -3634,10 +3634,7 @@ private void SubmitProfitTargetInternal()
             if (tracing)
                 Print("submitProfitTargetInternal(orderEntry " + orderEntry != null ? orderEntry.ToString() : "null" + ")");
 
-
-
-
-            if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode)

```

Comment:
This is required - IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode use the same paths a lot of the time-